TestResultSummaryService/parsers/TestBenchmarkParser.js

```diff
@@ -41,7 +33,8 @@ class TestBenchmarkParser extends Parser {
             for(let result of results) {
                 if (this.canParse(result.testName)) {
                     let {benchmarkName, benchmarkVariant} = this.getBenchmarkInfo(result.testName);
-                    const {testData} = Utils.parseOutput(benchmarkName, benchmarkVariant, result.testOutput);
+                    const benchmarkParserKey = Utils.getBenchmarkParserKey(result.testName);
+                    const {testData} = Utils.parseOutput(benchmarkParserKey, result.testOutput);
```

Comment:
In `parseOutput()`, we can just return `testData` (not the object). There is no need to make an object and de-object it.